mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -238,8 +240,48 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
     }
   ]
 
+  // Asset sort bottom sheet list
+  const assetValue = 'Asset value'
+  const [assetSortType, setAssetSortType] = useState(assetValue) // to display selected sorted type text
```

Comment:
This could potentially be prone to user error mistake. Can we change to enum?

Reply:
a string type will be passed to the bottomsheet modal when we select the sort type